### PR TITLE
AGS: Add language detection for games/demos E-G

### PR DESCRIPTION
**Pull request by tag2015**

### CODE REVIEW

File: engines/ags/detection_tables.h [hunk 1]
```diff
@@ -3601,14 +3601,14 @@ const AGSGameDescription GAME_DESCRIPTIONS[] = {
...
 	DEMO_ENTRY_EN("dysmaton", "Dysmaton.ags", "1e0cda052a371ebf6aa0bd62c0e754b9", 138033300),
-	DEMO_ENTRY("earlbobby3", "demo.exe", "0e32c4b3380e286dc0cea8550f1c045e", 4221725),
-	DEMO_ENTRY("echointheclouds", "Echo in the Clouds.exe", "2321081ba51d1869ebe11b9ba126911b", 57717828),
-	DEMO_ENTRY("eerieblue", "EerieBlue.exe", "bacdba3c759a861e899e6b0adcbb8bd5", 96605122),
-	DEMO_ENTRY("eerieblue", "EerieBlue.exe", "bacdba3c759a861e899e6b0adcbb8bd5", 96647570),
-	DEMO_ENTRY("emmaroide", "emma roide.exe", "aefd91a131817036d224fe030e406c6e", 24919443),
-	DEMO_ENTRY("enterthestory", "Enter The Story.exe", "0514661a4ba6772cf0db0cf839fc7003", 19589742),
-	DEMO_ENTRY("everlight", "Everlight.exe", "0500aacb6c176d47ac0f8158f055db83", 17597037),
-	DEMO_ENTRY("everythingm", "TinyMika.exe", "66adc3431f0626a2c6e80f31a21f0981", 3529496),
+	DEMO_ENTRY_EN("earlbobby3", "demo.exe", "0e32c4b3380e286dc0cea8550f1c045e", 4221725),
```
Comment (bluegr) [inline engines/ags/detection_tables.h:3604]: This game has been released in three languages (English, German and Italian):
https://www.adventuregamestudio.co.uk/site/games/game/1375-earl-bobby-is-looking-for-a-loo/

Comment (bluegr) [reply engines/ags/detection_tables.h:3604]: Edit: Nevermind, this is the demo - the full version has been entered correctly below

File: engines/ags/detection_tables.h [hunk 2]
```diff
@@ -4527,249 +4529,250 @@ const AGSGameDescription GAME_DESCRIPTIONS[] = {
...
+	GAME_ENTRY("errand", "errand.exe", "2e6b104176f310e7fb507183c443edfe", 4404813), // v1.2  Multilang
+	GAME_ENTRY("errand", "Errand.exe", "8212e96af620f6e02a37fdf583f79751", 4425035),  // v1.3  Windows Multilang
+	GAME_ENTRY("errand", "Errand.exe", "8212e96af620f6e02a37fdf583f79751", 4421187),  // v1.3  Alt/Linux Multiang
+	GAME_ENTRY_EN("escape", "Escape.exe", "9cf51833e787cc919837d9a8bd8fc14c", 2182153),
+	GAME_ENTRY_EN("escapefromasmallroom", "game1.exe", "0b7529a76f38283d6e850b8d56526fc1", 803029),
+	GAME_ENTRY_EN("escapefromevergreenforest", "EFEF.exe", "2e58934bbe98335f33c20a23a6c6135a", 23855632),
+	GAME_ENTRY_EN("escapefromlurrilous", "EFL.exe", "4bcbc24015114752b3c7971128704689", 2756979),
+	GAME_ENTRY_EN("escapefromterrorbay", "eftb.exe", "665fe5817e020e6a81849429e93662e4", 4917879),
+	GAME_ENTRY_EN("escapefromthechaoticcity", "ChaoticCity.exe", "ff3358d8f2726d544aadfde4f1ec8407", 93118664),
+	GAME_ENTRY_EN("escapefromthegarage", "Escape.exe", "3b7cceb3e4bdb031dc5d8f290936e94b", 962798),
```
Comment (bluegr) [inline engines/ags/detection_tables.h:4583]: This is a game that was released in Spanish only. I believe that this entry should be updated accordingly.
https://www.adventuregamestudio.co.uk/site/games/game/742-escape-from-the-garage/

Comment (tag2015) [reply engines/ags/detection_tables.h:4583]: The description on the ags website is outdated, or refers to some other game... the version that gets detected by scummvm is completely in english. It's available here if you want to try it https://archive.org/details/escape-from-the-garage

Comment (bluegr) [reply engines/ags/detection_tables.h:4583]: The archive.org page points to the same AGS page, which mentions that the game is in Spanish

https://www.adventuregamestudio.co.uk/site/games/game/742

Comment (tag2015) [reply engines/ags/detection_tables.h:4583]: You can also download the game directly from the archive.org page (on the right, "zip" under download options), if you try it you'll see that it's in english. As I said before the AGS page is wrong, I don't know why.

Comment (bluegr) [reply engines/ags/detection_tables.h:4583]: OK. Thanks for the heads up!

**Comment by bluegr** [comment]
Thanks for your work! After reviewing almost all of these entries, the changes done to them are correct.

Merging

**Pull request merged**